docs/api.md

```diff
@@ -573,8 +565,8 @@ const main = async () => {
   // Offset current block height by 42
   await setBlockOffset(42);
 
-  const blockOffset = await getBlockOffset();
-  console.log({ blockOffset });
+  const [blockOffset, err] = await getBlockOffset();
+  console.log(blockOffset, err);
```

Comment:
Same here.

Reply:
Changed into the following format: `console.log({ blockOffset }, { err })`. Please let us know if this isn't the format you are looking for.